truffle/com.oracle.truffle.tools.debug.shell/src/com/oracle/truffle/tools/debug/shell/client/REPLRemoteCommand.java

```diff
@@ -579,6 +579,7 @@ void processReply(REPLClientContext context, REPLMessage[] replies) {
             } else {
                 context.displayFailReply(replies[0].get(REPLMessage.DISPLAY_MSG));
             }
+            throw new REPLContinueException();
```

Comment:
I don't know. This change is from original Martin @entlicher commit - which was part of #137 pull request.